projects/02-dataverse-explorers/README.md

```diff
@@ -0,0 +1,446 @@
+con técnicas de generación de imágenes como `Stable Diffusion` para generar
+imágenes.
+
+Además **construirás una _página web_ para visualizar un
+_conjunto (set) de datos_** que se adecúe a lo que descubras que tu usuario
+necesita.
+
+Como entregable final tendrás una página web que permita **visualizar la data,
+filtrarla, ordenarla y hacer algún cálculo agregado**. Con cálculo agregado
+nos referimos a distintos cálculos que puedes hacer con la data para mostrar
+información aún más relevante para los usuarios (promedio, el valor máximo
+o mínimo, etc).
+
+En este proyecto nos iremos familiarizando con el uso de herramientas de IA
+(inteligencia artificial) como chatGPT o herramientas de [Stable Diffussion](https://es.wikipedia.org/wiki/Stable_Diffusion).
+Así que tu tarea en este proyecto
+será pedirle a la IA un set de datos en formato [JSON](https://www.json.org/json-es.html)
+para que lo puedas consumir en tu proyecto y mostrar la data en una página web.
+
+La temática será a tu gusto, por ejemplo, pueden ser personajes importantes en la
+historia, países, películas... etc. La estructura de este archivo JSON deberá ser
+la siguiente:
+
+* `name:` Nombre del dato a dibujar en el front.
+* `short-description:` Descripción corta sobre el dato.
+* `img:` URL de la imagen. Esta imagen será generada a través del método Stable
+  diffussion. Una vez generada la imagen, ésta será descargada y guardada en
+  una carpeta del proyecto. La URL de esta imagen será agregada manualmente al
+  JSON generado por la AI. Para generar esta imagen, te recomendamos esta
+  [herramienta](https://huggingface.co/spaces/nota-ai/compressed-stable-diffusion).
+* `number:` Un valor numérico para el cálculo agregado (por ejemplo,
+  puntuaciones, cantidades, etc).
+* `number-description:` Descripción de lo que representa el número anterior.
+* `long-description:` Descripción larga sobre el dato.
+* `extras:` Es importante que contenga un objeto interior con al menos dos
+  valores, por ejemplo si tu data es sobre países puedes crear este objeto
+  interno con los siguientes valores:
+  - Idioma oficial
+  - Población
+
+  Los nombres de la propiedades internas del objeto `extras` las puedes
+  elegir tu.
+
+Un ejemplo de data, según los requisitos anteriores podría ser:
+
+```json
+{
+  "name": "Ada Lovelace",
+  "short-description": "Primera programadora de la historia",
+  "img": "URL_DE_LA_IMAGEN_GENERADA",
+  "number": 1843,
+  "number-description": "año de nacimiento",
+  "long-description": "Augusta Ada King, Condesa de Lovelace, fue una
+    matemática y escritora británica. Es conocida por su trabajo en la
+    máquina analítica de Charles Babbage, donde escribió el primer
+    algoritmo destinado a ser procesado por una máquina. Por esta razón,
+    es considerada la primera programadora de la historia.",
+  "extras": {
+    "country-of-birth": "Inglaterra",
+    "country of death": "Inglaterra",
```

Comment:
`country-of-death`

Reply:
Si ya tenemos las "statistics" como un objeto interno, creo que este campo "extras" estaría de más. Lo voy a eliminar